src/Chip/Chip.js

```diff
@@ -0,0 +1,168 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+import DeleteIcon from '../svg-icons/navigation/cancel';
+
+export const styleSheet = createStyleSheet('Chip', (theme) => {
+  const { palette, shadows, transitions } = theme;
+  return {
+    root: {
+      fontSize: 13,
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      height: 32,
+      outline: 'none',
+      border: 0,
+      color: palette.getContrastText(palette.grey[300]),
+      backgroundColor: palette.grey[300],
+      boxShadow: shadows[0],
```

Comment:
I think it was the default for a dynamic style, so is no longer needed. Removed.